lib/exercism/guest.rb

```diff
@@ -3,6 +3,7 @@
 class Guest
   def id
   end
+  alias_method :flipper_id, :id
```

Comment:
It is a little confusing. I could as easily alias `username` and use the value `"guest"` instead of `nil`. If https://github.com/guest signs in, nothing will break ¯\_(ツ)_/¯.